client/src/components/SignUpHeader/useStyles.ts

```diff
@@ -0,0 +1,24 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+  authHeader: {
+    alignSelf: 'flex-end',
+    justifyContent: 'flex-end',
+    alignItems: 'center',
+    display: 'flex',
+    flexWrap: 'wrap',
+  },
+  link: { textDecoration: 'none' },
+  accBtn: {
+    width: 170,
+    height: 54,
+    borderRadius: theme.shape.borderRadius,
+    filter: 'drop-shadow(0px 2px 6px rgba(74,106,149,0.2))',
+    backgroundColor: '#f04826',
+    color: '#ffffff',
+    boxShadow: 'none',
+    marginRight: 35,
+  },
+}));
```

Comment:
This is repeated, it'd be better to move this to a shared directory where both Login and Signup can utilize the styling.

Reply:
Ethan, I had left these in their own separate directories because the button styles are different and I assume we will be refactoring the entire navbar to be one component in the future. Let me know if I should still merge the authHeader section, though.

Reply:
This is good! Thank you 👍🏻